code/__DEFINES/food.dm

```diff
@@ -37,3 +37,6 @@
 #define FOOD_MEAT_HUMAN 50
 #define FOOD_MEAT_MUTANT 100
 #define FOOD_MEAT_MUTANT_RARE 200
+
+
+#define IS_EDIBLE(O) istype(O, /obj/item/reagent_containers/food/snacks) || O.GetComponent(/datum/component/edible)
```

Comment:
```suggestion
#define IS_EDIBLE(O) (istype(O, /obj/item/reagent_containers/food/snacks) || O.GetComponent(/datum/component/edible))
```

You should wrap the expression in brackets so that in a situation where it is used like this it won't act fucky:
`IS_EDIBLE(a_reagent_container) && 0`